app/src/main/res/values/dimens.xml

```diff
@@ -112,4 +112,7 @@
     <dimen name="feed_group_carousel_top_bottom_margin">2dp</dimen>
     <dimen name="feed_group_carousel_between_items_margin">4dp</dimen>
 
+    <dimen name="search_suggestion_text_size">16sp</dimen>
+    <dimen name="search_suggestion_helper_text_size">14sp</dimen>
```

Comment:
What did you base the text size on?

Reply:
Nothing specific, I tried some numbers until it was looking good.